bindings/pyroot/pythonizations/test/rdf_filter.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import ROOT
+import numpy as np
+
+# These imports are specific to test_with_dtypes
+import random 
+import pandas as pd
+import os
+from helper import *
+
+class PyFilter(unittest.TestCase):
+    """
+    Testing Pythonized Filters of RDF
+    """
+
+    def test_with_dtypes(self):
+        """
+        Tests the pythonized filter with all the tree datatypes and 
+        """
+        CreateData()
+        rdf = ROOT.RDataFrame("TestData", "./TestData.root")
+        test_cols =[str(c) for c in rdf.GetColumnNames()]
+        df = pd.DataFrame(rdf.AsNumpy())
+        for col_name in test_cols:
+            filter = "filter_" + TYPE_TO_SYMBOL[col_name] # string of filter name
+            func = eval(filter); # function of the filter
+            x = df[col_name].mean(); x = float(x);
```

Comment:
To help you in removing the pandas dependency, you can use the `Mean` operation from RDF for these cases https://root.cern/doc/master/classROOT_1_1RDF_1_1RInterface.html#a685e954c0a81643a18dd17f831fb730f

Reply:
But you don't really need this, you can just decide for some constant value to pass to all functions, so you don't spend extra time in computing the mean of the column for every column

Reply:
Also it's currently casting the mean to `float` in any case, so there's already some narrowing issue going on